timetracker/timetracker/urls.py

```diff
@@ -15,11 +15,24 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpRequest
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	visitorName = request.GET.get("name", False)
```

Comment:
Another little one: `request.GET.get("name")` will return None (a falsey value) by default, so there's no need to explicitly add the `False`.